quick_check.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Very quick compilation check
+cd /home/daniel/work/puzzgameFlutter
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)